tor/Dockerfile

```diff
@@ -13,10 +13,39 @@ RUN \
     apk add --no-cache \
         coreutils=9.3-r1 \
         openssl=3.1.4-r1 \
-        tor=0.4.8.9-r0
+        tor=0.4.8.9-r0 \
+        go=1.20.10-r0 \
+        git=2.40.1-r0 \
+        ca-certificates=20230506-r0 \
+        libcap=2.69-r0
+
+# Install obfs4proxy transport client
+RUN git clone https://github.com/Yawning/obfs4.git /go/obfs4/
```

Comment:
- Please clone a specific version
- Add the version as a variable and configure renovate to update it.

Reply:
Fixed